File: IT/Strutture dati/Esercizi/Python/esercizio2.py
# file esercizio2.py

from __future__ import print_function
from sys import stdout
from os import linesep

nums = [1, 3, 5, 7, 9]

i = 0

while i < len(nums):
	print(nums[i], end=" ")
	i += 1

stdout.write(linesep)

File: IT/Strutture dati/Esercizi/C++/esercizio2.cpp
/* file esercizio2.cpp */

#include <iostream>

using std::cout;
using std::endl;

int main()
{
	int nums[] = {1, 3, 5, 7, 9};

	int i = 0;

	while (i < sizeof(nums) / sizeof(int)) {
		cout << nums[i] << " ";
		i++;
	}

	cout << endl;
}

